package.json

```diff
@@ -5,9 +5,13 @@
   "main": "index.js",
   "scripts": {
     "build": "rm -rf build && babel -d ./build ./src -s",
+    "start": "node ./build/index.js",
```

Comment:
we need to first run the build then run the server i.e `"npm run build && node ./build/index.js"`

Reply:
done